lib/insights/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -4,22 +4,32 @@ module Common
       module ApplicationControllerMixins
         module ExceptionHandling
           DEFAULT_ERROR_CODE = 400
+          PUNDIT_UNAUTHORIZED_ERROR = "Pundit::NotAuthorizedError".freeze
```

Comment:
What about a `pundit_exception_handling` class to define these custom constants in.